src/gui/CMakeLists.txt

```diff
@@ -216,6 +219,11 @@ WidgetDataNode.C
 mini3D.C
 )
 
+if(VISIT_ANARI)
```

Comment:
```suggestion
if(HAVE_ANARI)
```

Reply:
Added `set(HAVE_LIBANARI true)` like OSPRay did. Just having `add_definitions(-DHAVE_ANARI)` doesn't work with the CMake expression.